meilisearch-http/src/analytics/segment_analytics.rs

```diff
@@ -210,10 +210,20 @@ impl Segment {
                     "server_provider": std::env::var("MEILI_SERVER_PROVIDER").ok(),
             })
         });
-        let infos = json!({
-            "env": opt.env.clone(),
-            "has_snapshot": opt.schedule_snapshot,
-        });
+        let infos = {
+            let db_path = opt.db_path != PathBuf::from("./data.ms");
+            let import_dump = opt.import_dump.is_some();
+            let dumps_dir = opt.dumps_dir != PathBuf::from("dumps/");
+            let import_snapshot = opt.import_snapshot.is_some();
+            let http_addr = opt.http_addr != "127.0.0.1:7700";
```

Comment:
Can you add a comment explaining what you are really doing here, please?

Reply:
Done